File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/map/adapter/SimpleInfoWindowAdapter.java
package com.skedgo.tripkit.ui.map.adapter;

import android.view.View;
import com.google.android.gms.maps.GoogleMap;
import com.google.android.gms.maps.model.Marker;

public class SimpleInfoWindowAdapter implements GoogleMap.InfoWindowAdapter {
  @Override
  public View getInfoWindow(Marker marker) {
    // Fall back to default implementation.
    return null;
  }

  @Override
  public View getInfoContents(Marker marker) {
    return null;
  }
}

File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/map/adapter/SimpleInfoWindowAdapter.kt
package com.skedgo.tripkit.ui.map.adapter

import android.view.View
import com.google.android.gms.maps.GoogleMap.InfoWindowAdapter
import com.google.android.gms.maps.model.Marker

open class SimpleInfoWindowAdapter : InfoWindowAdapter {
    override fun getInfoWindow(marker: Marker): View? {
        // Fall back to default implementation.
        return null
    }

    override fun getInfoContents(marker: Marker): View? {
        return null
    }
}